public/app/plugins/datasource/cloud-monitoring/datasource.ts

```diff
@@ -113,7 +113,7 @@ export default class CloudMonitoringDatasource extends DataSourceWithBackend<
     const queries = options.targets;
 
     if (!queries.length) {
-      return lastValueFrom(of({ results: [] }));
+      return lastValueFrom(of<{ [key: string]: any }>({}));
```

Comment:
@grafana/partner-datasources plz review this 🙏

if you can find a way to type this without the `any`, it would be appreciated 😌

Reply:
On second look, this probably seems to be a bigger runtime changes than desired 🤔